### PR TITLE
Add links to voters list

### PR DESCRIPTION
**Pull request by evanmhm**
[🥒](https://www.compart.com/en/unicode/U+1F952)

### CODE REVIEW

**Comment by PlasmaPower** [comment]
To clarify, this links usernames in the interactive voter list.

**Review by andrewda — APPROVED** [review]

**Comment by PlasmaPower** [comment]
Currently, this might have the potential for an XSS. Wrap the URL in quotes, then call `escape` on the username. @andrewda I suggest you revoke your review (post a neutral review after it) until this is fixed.

**Comment by PlasmaPower** [comment]
Actually, there already was an XSS. I guess I'm :+1: on this change, but please also fix the XSS.

**Comment by PlasmaPower** [comment]
I'm not sure that's possible with a GitHub username, but better safe than sorry. Maybe some time of weird space messes things up too.

File: server/static/js/chaos.js [hunk 1]
```diff
@@ -161,7 +161,7 @@ request.onreadystatechange = function() {
 
       var tablehtml = "<table>";
       for (var i = 0; i < list.length && i < 20; i++) {
-        tablehtml += "<tr><td>" + list[i].names + "</td><td>" + list[i].votes + "</tr>";
+        tablehtml += "<tr><td><a href=\"https://github.com/" + list[i].names + "\">" + list[i].names + "</a></td><td>" + list[i].votes + "</tr>";
```
Comment (PlasmaPower) [inline server/static/js/chaos.js]: Could you also replace `list[i].names` with `escape(list[i].names)`?

Comment (PlasmaPower) [reply server/static/js/chaos.js]: Oops that's for URIs. Don't do that.

Comment (PlasmaPower) [reply server/static/js/chaos.js]: I'd go with this: https://stackoverflow.com/a/12034334/3352674

Comment (PlasmaPower) [reply server/static/js/chaos.js]: NVM that's correct for the URL.

**Review by PlasmaPower — APPROVED** [review]
Looks good to me now.

**Comment by PlasmaPower** [comment]
Looks like GH usernames are alphanumeric, so there's no reason for worry. Sorry for the confusion!

**Pull request merged**